_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,209 @@
+
+By including the segwit code in Bitcoin Core 0.13.0, users gain several advantages:
+
+1. **Easier upgrade to segwit:** The code differences ("diff") from this release to a subsequent release that includes segwit will be small.  This allows users who modify their version of Bitcoin Core to easily convert any modifications they make to Bitcoin Core 0.13.0 to the release that contains segwit (which is expected to be 0.13.1).
+
+2. **Easier segwit testing:** Although this release won't run segwit code on mainnet, it does run the code on testnet and in regression testing mode (regtest), which makes it easy for developers, administrators, and testers to use segwit in a safe environment with a version of Bitcoin Core that will be very close to the first version that is ready for miners to activate segwit.
+
+3. **Full integration with other features:** all the other features included in this release -- such as feefiltering, compact block relay, child-pays-for-parent mining, and official binaries for Linux on ARM -- are integrated with the segwit code and will probably be in production for two or more months before segwit activates, providing extra time for potential problems to be discovered through community review and
+testing.
+
+**More information:**
+
+- [Release notes][release notes segwit]
+
+- [Segregated Witness: the next
+  steps](https://bitcoincore.org/en/2016/06/24/segwit-next-steps/)
+  BitcoinCore.org blog post
+
+- [BIP141: Segregated witness (consensus
+  layer)](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki),
+  technical information about segwit and where the activation parameters for segwit will be published
+
+## Compact block relay
+
+Prior to Bitcoin Core 0.13.0, a running full node would (by default) receive many transactions twice:
+
+1. Before the transaction was confirmed, as an individual transaction being relayed across the network.
+
+2. After the transaction was confirmed, as part of a group of transactions contained in a newly-mined block being relayed across the network.
+
+There's no need for the node to receive the transaction a second time if it still has the first copy, so compact block relay (BIP152) allows a node to receive from its peers an ordered list of what transactions are included in a new block so that the node can use the transactions it has already received to partly or fully reconstruct the transactions part of the block for itself.  If the node doesn't receive all the transactions it needs to fully reconstruct the block, it requests the missing transactions from its peers, and then uses them to complete the block.
+
+![Compact Blocks diagram](https://raw.githubusercontent.com/bitcoin/bips/master/bip-0152/protocol-flow.png)
+
+Compact blocks provides two very important benefits to the network:
```

Comment:
+ "and opportunistically sending small amounts of data to peers that ask for it"?